payment_payfip/controllers/main.py

```diff
@@ -2,63 +2,63 @@
 import pprint
 import werkzeug
 
-from odoo import _, http
+from odoo import http, exceptions
 from odoo.http import request
 
-from odoo.addons.payment.models.payment_acquirer import ValidationError
+# from odoo.addons.payment.models.payment_acquirer import ValidationError
 
 _logger = logging.getLogger(__name__)
 
 
 class PayFIPController(http.Controller):
-    @http.route('/payment/payfip/pay', type='http', auth='none', methods=['GET', 'POST'], csrf=False)
+    @http.route("/payment/payfip/pay", type="http", auth="none", methods=["GET", "POST"], csrf=False)
```

Comment:
Par convention, nous gardons les variables en simple quote et les textes en doubles quotes.

Si c'est traduit = double quotes
Si cela ne peut pas être modifié sans "casser" le fonctionnement = simple quote

```suggestion
    @http.route('/payment/payfip/pay', type='http', auth='none', methods=['GET', 'POST'], csrf=False)
```